bin/test-cleanup

```diff
@@ -80,13 +85,13 @@ if [[ ${namespaces_controlplane[*]}   || \
 fi
 
 echo "cleaning up viz extension resources, if present [${k8s_context}]"
-linkerd viz uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
+"$linkerd_path" viz uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
 
 echo "cleaning up multicluster resources, if present [${k8s_context}]"
-linkerd mc uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
+"$linkerd_path" mc uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
```

Comment:
It appears `linkerd mc uninstall` needs to happen before the control plane is deleted, otherwise that returns 
```
$ bin/linkerd mc uninstall
Error: you need Linkerd to be installed in order to install multicluster addons
```